### ISSUE TITLE
Default Constraints

### ISSUE THREAD

**Issue by bc-nf** [Epaminaidos/reorder-columns#1]
Noticed you blocked out user named constraints and default constraints. 
default_constraints.[type] != 'D' 
default_constraints.is_system_named != 0

I took these out and it seemed to work, but wondering why these are filtered?

**Comment by AndreasHoffmann2** [comment]
Tldr:
There is no particular reason. If it works and you understand these values, I would appreciate a PR.

Long version:
The script works by dropping and recreating the table, indexes, views, etc. In order to recreate these, the script needs to convert the metadata from the sys-tables into the required SQL-statements.
If the script deletes an object, whose metadata it does not fully understand, the metadata will not be added to the SQL-statement for the recreation. This there would be unwanted changes in the database after the execution.

As you might already guessed, this happened to us even in prod. Thus I added these restrictions: the script rejects all metadata that I did not know and did not need yet.

If you need it, please feel free to remove this check. If you open a PR, please add a short comment, why these checks can be removed without changing the database unintentionally. E.g with an example, how the object was created.

**Comment by bc-nf** [comment]
Thanks, will do. Appreciate the quick and thorough explanation on code you wrote years ago. I can't usually remember the reason for things on code I wrote months ago. :)
